.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  unit:
+    docker:
+      - image: hashicorpdev/consul-helm-test:0.2.0
```

Comment:
Is this created by `test/docker/Test.dockerfile`? If so can you put a comment at the top of this file about where the `hashicorpdev/consul-helm-test` image comes from.

Reply:
Yep it is, I've added a comment 🙏